src/main/java/com/baidu/hugegraph/formatter/KGDumper/ComputeSign.java

```
@@ -0,0 +1,105 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.formatter.kgdumper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ComputeSign {
+
+    private Map<String, Integer> tradeValue2Num;
+    private Map<String, String> entPlain2Id;
+
+    private String inputEncoding;
+    private String outputEncoding;
+
+    public ComputeSign(String tradeValueFile, String inputEncode, String outputEncode) throws IOException {
+        File file = new File(tradeValueFile);
+        Long fileLen = file.length();
+        byte[] fileBytes = new byte[fileLen.intValue()];
+        try {
+            FileInputStream in = new FileInputStream(file);
+            in.read(fileBytes);
+            in.close();
+        } catch (FileNotFoundException e) {
+			throw new FileNotFoundException("tradeValueFile not found:" + tradeValueFile);
+        } catch (IOException e) {
+            throw e;
+        }
+        tradeValue2Num = new HashMap<String, Integer>();
+        try {
+            String fileContent = new String(fileBytes, inputEncode);
+            String[] lines = fileContent.split("\n");
+            for (int i = 0; i < lines.length; i++) {
+                String[] cols = lines[i].split("\t");
+                if (cols.length >= 2) {
+                    tradeValue2Num.put(cols[0], Integer.valueOf(cols[1]));
+                }
+            }
+        } catch (UnsupportedEncodingException e) {
+            System.err.println("The OS does not support GBK");
+            e.printStackTrace();
```

Don't swallow exception, throw RuntimeException instead